infrastructure/terraform/accounts/account.tf

```
@@ -0,0 +1,122 @@
+# Parent Platform account
+
+module "aws_account" {
+  source = "./modules/account/aws"
+
+  # 4 hours
+  max_session_duration_in_seconds = "14400"
```

Note: terraform allows you to do maths expressions, so you could do

```suggestion
  max_session_duration_in_seconds = 4 * 60 * 60
```